src/dotnet/UI.Blazor/Components/Modal/BlazoredModal/CascadingBlazoredModal.razor

```diff
@@ -0,0 +1,35 @@
+﻿@namespace Blazored.Modal
```

Comment:
Let's move this thing to service & call it `BlazoredModalUI` or something.

Reply:
Removed this component